### PR TITLE
refactor(phrases): update fr translation

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- make `translate create` compatible with partial-translated phrases
- update fr translation

## Testing

CI

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/phrases/src/locales/fr/translation/admin-console/connectors.ts [hunk 1]
```diff
@@ -37,54 +38,55 @@ const connectors = {
...
     target_tip:
-      'The value of “IdP name” can be a unique identifier string to distinguish your social identifies. This setting cannot be changed after the connector is built.', // UNTRANSLATED
+      'La valeur de "nom de l\'IdP" peut être une chaîne d\'identificateur unique pour distinguer vos identifiants sociaux. Ce paramètre ne peut pas être modifié une fois que le connecteur est créé.',
     target_tooltip:
-      '"Target" in Logto social connectors refers to the "source" of your social identities. In Logto design, we do not accept the same "target" of a specific platform to avoid conflicts. You should be very careful before you add a connector since you CAN NOT change its value once you create it. <a>Learn more.</a>', // UNTRANSLATED
+      "«Cible» dans les connecteurs sociaux de Logto fait référence à la «source» de vos identités sociales. Dans la conception de Logto, nous n'acceptons pas la même «cible» d'une plateforme spécifique pour éviter les conflits. Vous devriez être très attentif avant d'ajouter un connecteur car vous NE POUVEZ PAS changer sa valeur une fois que vous l'avez créé. <a>En savoir plus.</a>",
```
Comment (gao-sun) [inline packages/phrases/src/locales/fr/translation/admin-console/connectors.ts:63]: not sure should we keep the original english in `<a>`

**Review by simeng-li — APPROVED** [review]
LGTM

File: packages/phrases/src/locales/fr/translation/admin-console/application-details.ts [hunk 2]
```diff
@@ -2,29 +2,29 @@ const application_details = {
...
   application_name: "Nom de l'application",
   application_name_placeholder: 'Mon App',
   description: 'Description',
   description_placeholder: 'Entrez la description de votre application',
-  authorization_endpoint: 'Authorization Endpoint',
+  authorization_endpoint: "Point de terminaison d'autorisation",
   authorization_endpoint_tip:
-    "Le point de terminaison pour effectuer l'authentification et l'autorisation. Il est utilisé pour <a>l'authentification</a> OpenID Connect.",
-  application_id: 'App ID',
+    "Le point de terminaison pour effectuer l'authentification et l'autorisation. Il est utilisé pour l'authentification OpenID Connect.",
```
Comment (gao-sun) [inline packages/phrases/src/locales/fr/translation/admin-console/application-details.ts:17]:
```suggestion
    "Le point de terminaison pour effectuer l'authentification et l'autorisation. Il est utilisé pour <a>l'authentification</a> OpenID Connect.",
```

File: packages/phrases/src/locales/fr/translation/admin-console/application-details.ts [hunk 3]
```diff
@@ -2,29 +2,29 @@ const application_details = {
...
+  application_id: "ID de l'application",
   application_id_tip:
-    'The unique application identifier normally generated by Logto. It also stands for “<a>client_id</a>” in OpenID Connect.', // UNTRANSLATED
-  application_secret: 'App Secret',
-  redirect_uri: 'Redirect URI',
-  redirect_uris: 'Redirect URIs',
+    "L'identifiant d'application unique généralement généré par Logto. Il signifie également « client_id » dans OpenID Connect.",
```
Comment (gao-sun) [inline packages/phrases/src/locales/fr/translation/admin-console/application-details.ts:20]:
```suggestion
    "L'identifiant d'application unique généralement généré par Logto. Il signifie également <a>client_id</a> dans OpenID Connect.",
```

File: packages/phrases/src/locales/fr/translation/admin-console/application-details.ts [hunk 4]
```diff
@@ -2,29 +2,29 @@ const application_details = {
...
+  application_secret: "Secret de l'application",
+  redirect_uri: 'URI de redirection',
+  redirect_uris: 'URIs de redirection',
   redirect_uri_placeholder: 'https://votre.site.com/app',
   redirect_uri_placeholder_native: 'io.logto://callback',
   redirect_uri_tip:
-    "L'URI de redirection après la connexion d'un utilisateur (qu'elle soit réussie ou non). Voir OpenID Connect <a>AuthRequest</a> pour plus d'informations.",
+    "L'URI de redirection après la connexion d'un utilisateur (qu'elle soit réussie ou non). Voir OpenID Connect AuthRequest pour plus d'informations.",
```
Comment (gao-sun) [inline packages/phrases/src/locales/fr/translation/admin-console/application-details.ts:27]:
```suggestion
    "L'URI de redirection après la connexion d'un utilisateur (qu'elle soit réussie ou non). Voir OpenID Connect <a>AuthRequest</a> pour plus d'informations.",
```

**Pull request merged**